doc/nrf/ug_zigbee_qsg.rst

```diff
@@ -99,8 +99,8 @@ Software requirements
 For this quick start guide, we will install the following software:
 
 * Toolchain Manager - An application for installing the full |NCS| toolchain.
-* Visual Studio Code (VS Code) - The recommended IDE for the |NCS|.
-* |VSC| - Extension for VS Code that allows you to develop applications for the |NCS|.
+* |VSC| (VS Code) - The recommended IDE for the |NCS|.
```

Comment:
```suggestion
* Microsoft's Visual Studio Code - The recommended IDE for the |NCS|.
```

This is actually not the correct shortcut. |VSC| is actually a shortcut for "nRF Connect for Visual Studio Code".